tests/test_scripts/test_decorator.py

```diff
@@ -22,22 +22,38 @@ def test_init_ee() -> None:
 
 def test_catch_errors() -> None:
     """Check the catch error decorator."""
+    # create an external alert to test the wiring
+    alert = sw.Alert()
+
     # create a fake object that uses the decorator
     class Obj:
         def __init__(self):
             self.alert = sw.Alert()
             self.btn = sw.Btn()
 
-            self.func1 = sd.catch_errors(alert=self.alert)(self.func)
-            self.func2 = sd.catch_errors(alert=self.alert, debug=True)(self.func)
+        @sd.catch_errors()
+        def func0(self, *args):
+            return 1 / 0
+
+        @sd.catch_errors(alert=alert)
+        def func1(self, *args):
+            return 1 / 0
 
-        def func(self, *args):
+        @sd.catch_errors(debug=True)
+        def func2(self, *args):
             return 1 / 0
 
     obj = Obj()
 
-    obj.func1()
+    # should return an alert error in the the self alert widget
+    obj.func0()
     assert obj.alert.type == "error"
+
+    # should return an alert in the external alert widget
+    obj.func1()
+    assert alert.type == "error"
```

Comment:
alert type will remain `error` after the previous test, so this error is not properly tested. I think we should: create this object as fixture and rest both scenarios and/or add the `self.alert.type = "info"` to the reset method in the alert component.

Reply:
nice catch, I prefer to revert to info type on reset (I think it's a bug in the first place)

Reply:
I think solved in https://github.com/12rambau/sepal_ui/pull/856/commits/5d86ad169502ae61460a1f4fe71b75142501ef71